neqo-udp/src/lib.rs

```diff
@@ -346,7 +348,9 @@ mod tests {
 
         let sender = socket()?;
         let receiver = socket()?;
-        let receiver_addr: SocketAddr = "127.0.0.1:0".parse().unwrap();
+        let receiver_addr: SocketAddr = "127.0.0.1:0"
+            .parse()
+            .map_err(|_| io::ErrorKind::InvalidInput)?;
```

Comment:
I don't get this change: you are testing, so an expectation is fine.  Especially for this case, where it should be infallible.